IFC4x3/Constants/s/STATION_3mZYci1kzBIfEoa0zx_hua.xml

```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="STATION_3mZYci1kzBIfEoa0zx_hua" Name="STATION" UniqueId="f08e29ac-06ef-4b4a-93b2-900f7bfabe24" Status="Deprecated" />
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="STATION_3mZYci1kzBIfEoa0zx_hua" Name="STATION" UniqueId="f08e29ac-06ef-4b4a-93b2-900f7bfabe24">
+	<Documentation>Used to represent a linear location given by the linear element being measured (the _IfcAlignment_ into which the _IfcReferent_ is nested), a station value or a station equation using _Pset_Stationing_ and optionally the method of measurement (_Pset_LinearReferencingMethod_). If a linear referencing method is specified for the station, it overrides any linear referencing method specified for the alignment.</Documentation>
```

Comment:
I am a bit skeptical referencing the psets in the description but since I do not see anyone else objecting, I guess it is fine.

Somehow the first sentence seems a bit confusing. Here a proposal:
```suggestion
	<Documentation>Represents a position linearly measured along the linear positioning element into which the _IfcReferent_ is nested. The position is described by a station value or a station equation using _Pset_Stationing_ and optionally the method of measurement (_Pset_LinearReferencingMethod_). If a linear referencing method is specified for the station, it overrides any linear referencing method specified for the alignment.</Documentation>
```

Reply:
1)  Concerning `.POSITION.` : Sergej writes "It could be used for modelling any known location that could be referenced by building elements.". I am fine with this, I just wanted to see a plausiable argument for a situation where it might be used. I withdraw the proposal to remove it.

2) Concerning the proposed text change for the .STATION. referent: Would it be posible to use the word "location" instead of "position", so we do not blend in the `.POSITION.`here? If so, the text would read as follows: 

`[STATION] Represents a linearly measured location  along the linear positioning element into which the _IfcReferent_ is nested. The location is described by a station value or a station equation using _Pset_Stationing_ and optionally the method of measurement (_Pset_LinearReferencingMethod_). If a linear referencing method is specified for the _STATION_ referent, ithen t overrides any linear referencing method specified for the alignment.`

Reply:
I have propose position because my impression was that location is more physical in nature than position which can also be an abstract description. I am, however, no expert on this and am completely fine to go with the experts.

Reply:
I updated according to proposal of @clfey above. If also .POSITION. doc is ok, I think that it is time for a merge.

Reply:
Made a slight adjustment to formatting and removed "then" from the last sentence.